queries/c/highlights.scm

```diff
@@ -170,4 +170,6 @@
 (preproc_params
   (identifier)) @parameter
 
+(attribute) @attribute
```

Comment:
This will match each attribute separately, but not the surrounding `[[` and `]]`.

If you want to match instance of `[[...]]` as a single highlight, I'd change this to `(attribute_declaration) @attribute`

Reply:
also changed `co_yield`/`co_return` to `@keyword.return`